packages/relic/lib/src/headers/special/cross_origin_resource_policy_header.dart

```diff
@@ -0,0 +1,48 @@
+part of '../headers.dart';
+
+/// A class representing the HTTP Cross-Origin-Resource-Policy header.
+///
+/// This header specifies the policy for sharing resources across origins.
+class CrossOriginResourcePolicyHeader {
+  /// The policy value of the header.
+  final String policy;
+
+  /// Constructs a [CrossOriginResourcePolicyHeader] instance with the specified value.
+  const CrossOriginResourcePolicyHeader(this.policy);
```

Comment:
We should validate that the values are correct here. Only the defined values seem to be valid. Or, we can make the constructor private?